tests/test_collection.py

```diff
@@ -4,129 +4,139 @@
 
 from c8.collection import StandardCollection
 from c8.exceptions import (
-    CollectionConfigureError,
     CollectionPropertiesError,
-    CollectionTruncateError,
     CollectionCreateError,
     CollectionListError,
     CollectionDeleteError,
 )
 from tests.helpers import assert_raises, extract, generate_col_name
 
 
-def test_collection_attributes(db, col, username):
+def test_collection_attributes(client, col, tst_fabric):
     assert col.context in ['default', 'async', 'batch', 'transaction']
-    assert col.username == username
-    assert col.db_name == db.name
-    assert col.name.startswith('test_collection')
+    assert col.tenant_name == client._tenant.name
+    fabric = client._tenant.useFabric(tst_fabric.name)
+    assert col.fabric_name == fabric.name
```

Comment:
you can remove the "fabric" var (line no. 18) it's not needed it is basically equal to tst_fabric.. so wherever you use the "fabric" var instead simply use tst_fabric